app/decorators/commands/decidim/consultations/admin/update_consultation_decorator.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# This decorator add suara permissions fields to update consultation command
+Decidim::Consultations::Admin::UpdateConsultation.class_eval do
+  def attributes
```

Comment:
Can we use the same technique as in https://github.com/gencat/participa-catalunya/blob/master/app/decorators/decidim/courses/controllers/admin/courses_controller_decorator.rb to avoid the decorator being affected from changes in the command?